Makefile

```diff
@@ -42,10 +42,13 @@ _test_docs:
 	pytest --nbmake "./docs"
 
 test_docs:
+	kill -9 `ps aux | grep "[d]ocs.examples._data_server" | awk '{print $$2}'`
```

Comment:
This line seems to be causing tests to fail. I suspect adding `|| true` at the end (like the line below) should fix it?